src/template.html

```diff
@@ -1,22 +1,19 @@
-<div class="dayparts">
-
+<div class="dayparts" ng-class="{reverse: options.reverse}">
     <table border="0" cellspacing="0" cellpadding="0">
         <tr>
-            <th></th>
-            <th ng-repeat="hour in hours">
-                <a ng-if="!options.disableColumnSelection" ng-click="selectHour(hour)" ng-show="hour !== 0">{{hour}}</a>
-                <span ng-if="options.disableColumnSelection">{{hour}}</span>
+            <th id="blank-header"></th>
+            <th ng-repeat="row in rows" ng-click="selectLine(row)">
+                <a ng-if="!options.disableColumnSelection">{{row.title}}</a>
             </th>
         </tr>
-        <tr ng-repeat="day in days">
-            <th>
-                <a ng-if="!options.disableRowSelection" ng-click="selectDay(day)">{{day.name}}</a>
-                <span ng-if="options.disableRowSelection">{{day.name}}</span>
+        <tr ng-repeat="col in columns">
+            <th ng-click="selectLine(col)">
+                <a ng-if="!options.disableRowSelection">{{col.title}}</a>
             </th>
-            <td ng-repeat="hour in hours" data-time="{{day.name}}-{{hour}}"></td>
+            <td ng-repeat="row in rows" data-time="{{getDayTitleHourValue(row, col)}}"></td>
         </tr>
     </table>
-
+  
     <button type="button" ng-click="reset()" ng-if="options.reset">Reset</button>
-
-</div>
\ No newline at end of file
+  
```

Comment:
You added trailing whitespace here. There's an editor function to automatically delete it you can probably enable, JSYK!